lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -385,6 +385,145 @@ def export_json(content_view_version, repositories)
       end
     end
 
+    class ImportCommand < HammerCLIForeman::Command
+      include HammerCLIForemanTasks::Async
+      include HammerCLIKatello::LocalHelper
+      include HammerCLIKatello::ApipieHelper
+
+      attr_accessor :export_tar_dir, :export_tar_file, :export_tar_prefix
+
+      desc _('Import a content view version')
+
+      command_name "import"
+
+      success_message _("Content view imported.")
+      failure_message _("Could not import the content view.")
+
+      option "--organization-id", "ORGANIZATION_ID", _("Organization numeric identifier")
+      option(
+        '--export-tar',
+        'EXPORT_TAR',
+        _("Location of export tar on disk")
+      )
+
+      validate_options do
+        option(:option_export_tar).required
+        option(:option_organization_id).required
+      end
+
+      build_options
+
+      class OrganizationIdParamSource
+        def initialize(command)
+          @command = command
+        end
+
+        def get_options(_defined_options, result)
+          if result['option_organization_id'].nil? && result['option_organization_name']
+            result['option_organization_id'] = @command.resolver.organization_id(result)
+          end
+          result
+        end
+      end
+
+      def option_sources
+        super << OrganizationIdParamSource.new(self)
+      end
+
+      def execute
+        unless File.exist?(options['option_export_tar'])
+          raise _("Export tar #{options['option_export_tar']} does not exist.")
+        end
+
+        self.export_tar_file = File.basename(options['option_export_tar'])
+        self.export_tar_prefix = @export_tar_file.gsub('.tar', '')
+        self.export_tar_dir = File.dirname(options['option_export_tar'])
+        untar_export
+
+        export_json = read_json
+
+        cv = content_view(export_json['name'], options['organization_id'])
```

Comment:
All the options are prefixed with "option_", so this should be `options['option_organization_id']`. It's currently passing nil to the API.